code/script.js

```diff
@@ -1,51 +1,285 @@
+          showMessage(`I'll watch Terminator 2 tonight!`, 'user')
+          inputWrapper.innerHTML = ''
+          setTimeout (() => lastReplyAngry('terminator2'), 1000)
+        })
+  
+        document.getElementById("djangoUnchained").addEventListener('click', () => {
+          showMessage(`I'll watch Django Unchained tonight`, 'user')
+          inputWrapper.innerHTML = ''
+          setTimeout (() => lastReplyAngry('djangoUnchained'), 1000)
+        })  
+    }
+  }
+}
+
+//The last reply from bot: The bot replies to the user's choice of films, and thanks the user for using the chat bot.
+const lastReplyHappy = (selection) => {
+  if (selection === 'happyFeet') {
+    showMessage(`Great pick! Happy feet is one of my all time favorites. Enjoy the movie!`, 'bot');
+    setTimeout (() => inputWrapper.innerHTML = `
+      <p>Thank you for using this bot!</p>
+    `, 1000)
+  }
+  else if (selection === 'singingInTheRain') {
+    showMessage(`Good choice! Singing in the Rain sure is 10/10!`, 'bot')
+    setTimeout (() => inputWrapper.innerHTML = `
+      <p>Thank you for using this bot!</p>
+    `, 1000)
+  }
+  else if (selection === 'grease') {
+    showMessage(`Great pick! I LOVE Grease!`, 'bot')
+    setTimeout (() => inputWrapper.innerHTML = `
+      <p>Thank you for using this bot!</p>
+    `, 1000)
+  }
+}
+
+const lastReplySad = (selection) => {
+  if (selection === 'titanic') {
+    showMessage(`Classic, maybe this time Jack will fit on the door!`, 'bot')
+    setTimeout (() => inputWrapper.innerHTML = `
+    <p>Thank you for using this bot!</p>
+    `, 1000)
+  }
+  
+  else if (selection === 'theNotebook') {
+    showMessage(`Great choice, make sure to bring napkins!`, 'bot')
+    setTimeout (() => inputWrapper.innerHTML = `
+    <p>Thank you for using this bot!</p>
+    `, 1000)
+  }
+
+  else if (selection === 'brokebackMountain') {
+    showMessage(`Nice, that's one of my overall favorites!`, 'bot')
+    setTimeout (() => inputWrapper.innerHTML = `
+    <p>Thank you for using this bot!</p>
+    `, 1000)
+  }
+}
+
+const lastReplyAngry = (selection) => {
+  if (selection === 'angerManagement') {
+    showMessage(`Good one, sounds like it could be needed!`, 'bot')
+    setTimeout (() => inputWrapper.innerHTML = `
+    <p>Thank you for using this bot!</p>
+    `, 1000)
+  }
+
+  else if (selection === 'terminator2') {
+    showMessage(`Great choice. Hasta la vista, baby!`, 'bot')
+    setTimeout (() => inputWrapper.innerHTML = `
+    <p>Thank you for using this bot!</p>
+    `, 1000)
+  }
+
+  else if (selection === 'djangoUnchained') {
+    showMessage(`Good movie, one of Tarantinos best works!`, 'bot') 
+    setTimeout (() => inputWrapper.innerHTML = `
+    <p>Thank you for using this bot!</p>
+    `, 1000)
+  }
 }
 
 // Set up your eventlisteners here
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+  const value = document.getElementById('name-input').value;
```

Comment:
Just wondering ourselves if we should avoid naming variables with the same name as properties that exist in javascript..(i.e. const value vs. .value) ?

Reply:
I agree that it would be less confusing if we would have chosen another name for it! 

However, it seems that this is widely used (for example in Technigo's own example in Sprint 1). So, I don't know, maybe we just have to get used to it 🤔